k6ext/panic.go

```diff
@@ -36,3 +44,20 @@ func Panic(ctx context.Context, format string, a ...interface{}) {
 	_ = p.Release()
 	_ = p.Kill()
 }
+
+type userFriendlyError struct{ err error }
+
+func (e *userFriendlyError) Unwrap() error { return e.err }
+
+func (e *userFriendlyError) Error() string {
+	switch {
+	default:
+		return e.err.Error()
```

Comment:
I'm curious why `default` is the first case. Is there a functional reason for it?

Reply:
No functional reason. Only as a hint that most of the time the default one will match.